api/v3/Case/Getdetailscount.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Case.getdetailscount API
+ *
+ * Provides a count of cases but properly respects filters unlike `getcount`.
+ * Please refer to C51-277 for more information.
```

Comment:
I think normally we try to keep ticket numbers out of the code since we never know when things will be moved around. People can check the commit history and get the reference there anyway.